content/en/users/datatransfer/WebFTS/_index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "WebFTS"
+linkTitle: "WebFTS"
+type: docs
+description: "Documentation related to EGI Data Transfer Web Interface"
+weight: 30
+---
+
+## Overview 
+
+The WebFTS interface offers end users a way to graphically execute and monitor file transfer between storages using different protocols.
+
+The full documentation is accessible at:
+
+https://fts3-docs.web.cern.ch/fts3-docs/docs/webfts/userguide.html
+
+with FAQ at:
+
+https://fts3-docs.web.cern.ch/fts3-docs/docs/webfts/faq.html
+
+
+### Credential Delegation
+
+The authentication with the  Storage Elements (SE) is done based on X509 certificate which needs to be installed on the browser. The Integration with EGI Check-in is under development and will be available soon in the production instance.
+
+A pop-up window appears when you choose the "My jobs" or "Submit a transfer" tabs. There, you will need to paste the private RSA key of your certificate. The private key WILL NOT BE TRANSMITTED ANYWHERE. It is only used locally (within the user's browser) to generate the proxies needed to have access to the FTS services.
```

Comment:
```suggestion
A pop-up window appears when you choose the "My jobs" or "Submit a transfer" tabs. There, you will need to paste the private RSA key of your certificate. The private key **WILL NOT BE TRANSMITTED ANYWHERE**. It is only used locally (within the user's browser) to generate the proxies needed to have access to the FTS services.
```

Reply:
If needed you may want to use an infobox for this. (not found of capitalized text).